legal-api/src/legal_api/services/business_details_version.py

```diff
@@ -594,6 +594,8 @@ def resolution_json(resolution_revision) -> dict:
     @staticmethod
     def business_revision_json(business_revision, business_json):
         """Return the business revision as a json object."""
+        if not business_revision:
+            return business_json
```

Comment:
can you add a comment that this was added specifically to address tombstone pipeline scenario